shub/image/init.py

```diff
@@ -22,42 +21,41 @@
 RUN python setup.py install
 """.format(docker_app_dir=DOCKER_APP_DIR)
 
-BASE_SYSTEM_DEPS = [
-    'telnet', 'vim', 'htop', 'strace', 'ltrace', 'iputils-ping', 'lsof'
-]
 BASE_PYTHON_DEPS = [
-    'BeautifulSoup==3.2.0', 'MySQL-python==1.2.3', 'Scrapy==1.0.3',
-    'Twisted==11.1.0', 'boto==2.28.0', 'guppy==0.1.10', 'hubstorage==0.19.1',
-    'ipython==3.2.1', 'lxml==3.4.4', 'numpy==1.6.1', 'pika==0.9.14',
-    'psycopg2==2.4.5', 'pymongo==2.6.3', 'queuelib==1.2.2',
-    'raven==5.0.0', 'requests==2.5.3', 'scrapely==0.12.0',
-    'scrapinghub==1.7.0', 'scrapylib==1.6.0', 'setproctitle==1.0.1',
-    'slybot==0.12.1', 'w3lib==1.13.0',
-    'scrapinghub-entrypoint-scrapy==0.6.0'
+    'BeautifulSoup==3.2.0',
```

Comment:
The requirements were outdated mainly because it's hard-coded in the sources. Wdyt if we move it to some `base-requirements.txt` and drop a reminder to update it from time to time to README? Though it strongly correlates with chosen base image,  having some doubts if it improves the process and not complicates it.